tests/contrib/django/test_django_snapshots.py

```diff
@@ -46,13 +40,14 @@ def daphne_client(snapshot):
 
     # Wait for the server to start up
     client.wait()
+    # send request
+    resp = client.request(method, path)
+
+    shutdown = client.get_ignored("/shutdown-tracer/")
+    assert shutdown.status_code == 200
+    proc.terminate()
```

Comment:
we should keep a try/finally for terminating the process, we _have_ to be sure to kill the subprocess.